app/src/main/java/fr/neamar/kiss/loader/LoadDocsPojos.java

```diff
@@ -0,0 +1,112 @@
+package fr.neamar.kiss.loader;
+
+import android.content.Context;
+import android.database.Cursor;
+import android.provider.MediaStore;
+
+import java.util.ArrayList;
+
+import fr.neamar.kiss.R;
+import fr.neamar.kiss.pojo.DocsPojo;
+
+import static android.provider.MediaStore.MediaColumns.DATA;
+
+public class LoadDocsPojos extends LoadPojos<DocsPojo> {
+
+    public LoadDocsPojos(Context context) {
+        super(context, "file://");
+    }
+
+    @Override
+    protected ArrayList<DocsPojo> doInBackground(Void... params) {
+        ArrayList<DocsPojo> docs = new ArrayList<>();
+
+        final String[] DOC_PROJECTION = {
+                MediaStore.Images.Media._ID,
+                MediaStore.Images.Media.DATA,
+                MediaStore.Files.FileColumns.MIME_TYPE,
+                MediaStore.Files.FileColumns.SIZE,
+                MediaStore.Images.Media.DATE_ADDED,
+                MediaStore.Files.FileColumns.TITLE
+        };
+
+        String[] selectionArgs = new String[]{".pdf", ".ppt", ".pptx", ".xlsx", ".xls", ".doc", ".docx", ".txt"};
```

Comment:
> ok we can make a list together…

Good :smile: 

> Because sometimes MIME can be null. For instance in my device MIME is null for a yy.xls file.

In that case I think we should have a list of both MIME types and their corresponding file extensions (similar to [`/etc/mime.types`](https://wiki.debian.org/MIME/etc/mime.types)). This way we'd also have a central place for mapping file extensions to MIME types for `launch()`.

Here's what I think we should do:

 * Ship a short, but complete list of document MIME types and file extensions as raw resource in `/etc/mime.types` format with KISS (the format is basically `[MIME-Type] <[File-extension] <[File-extension] <…>>>`, see the link above); this shouldn't require more than 1kiB of space
 * Parse this list on document provider startup
 * When examining a document entry:
     1. Check if it's MIME type matches any document type list entry, if so, store the entry's Pojo with this type
     2. Check if it's filename ends with any file extension of the document type list, if so, store the entry's Pojo with the MIME type associated with the file extension
     3. Otherwise ignore the entry
 * When launching a document from the launcher, then just use the MIME type previously stored in it's Pojo

I'd also gladly compile the final MIME type list for you once we've agreed on all the file types we want to support.

The current list looks pretty reasonable, except for `mp3` which is definitely *not* a document format. If anything that should be part of the media provider that we may eventually have one day… (#390)